<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1736</width>
    <height>827</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Frutiger LT Com</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="MCLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Motor control: focus and output</string>
   </property>
  </widget>
  <widget class="QLabel" name="FocusNTlabel">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>210</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Focus Nanotrak</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeinitializeButton">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>710</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Deinitialize APT</string>
   </property>
  </widget>
  <widget class="QAxWidget" name="MotorControl">
   <property name="geometry" stdset="0">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>561</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font" stdset="0">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale" stdset="0">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QLabel" name="OutputNTLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Output Nanotrak</string>
   </property>
  </widget>
  <widget class="QAxWidget" name="FocusNanotrak">
   <property name="geometry" stdset="0">
    <rect>
     <x>1160</x>
     <y>250</y>
     <width>561</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font" stdset="0">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale" stdset="0">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QAxWidget" name="MotorControlChip">
   <property name="geometry" stdset="0">
    <rect>
     <x>590</x>
     <y>540</y>
     <width>561</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font" stdset="0">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale" stdset="0">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QLabel" name="MCCLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>500</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Motor control: input and chipstage</string>
   </property>
  </widget>
  <widget class="QAxWidget" name="OutputNanotrak">
   <property name="geometry" stdset="0">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>561</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font" stdset="0">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale" stdset="0">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QLabel" name="InputNTLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Input Nanotrak</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InitializeButton">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>710</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Initialize APT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TrackAllButton">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>760</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Track all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LatchAllButton">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>760</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Latch all</string>
   </property>
  </widget>
  <widget class="QAxWidget" name="InputNanotrak">
   <property name="enabled" stdset="0">
    <bool>true</bool>
   </property>
   <property name="geometry" stdset="0">
    <rect>
     <x>590</x>
     <y>250</y>
     <width>561</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font" stdset="0">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale" stdset="0">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="OutputNTSerial">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>221</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>52435044</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputNTSerial">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>220</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>52368214</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MCSerial">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>510</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>50862650</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FocusNTSerial">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>220</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>52368224</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MCChipSerial">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>510</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>50862640</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>540</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QDoubleSpinBox" name="InputMotor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="minimum">
     <double>-150.000000000000000</double>
    </property>
    <property name="maximum">
     <double>150.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="OutputMotor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="minimum">
     <double>-150.000000000000000</double>
    </property>
    <property name="maximum">
     <double>150.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="MiddleStage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="minimum">
     <double>-150.000000000000000</double>
    </property>
    <property name="maximum">
     <double>150.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FocusMotor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="minimum">
     <double>-150.000000000000000</double>
    </property>
    <property name="maximum">
     <double>150.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input stage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Output stage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Middle stage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Focus</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Height middle stage</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="HeightMiddleStage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="minimum">
     <double>-50000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="MoveMotorButton">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>540</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Move motor</string>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="General">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>APT-Control: NanoTrak and linear motor</string>
    </property>
   </widget>
   <widget class="QLabel" name="General1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fraunhofer Heinrich-Hertz-Institute</string>
    </property>
   </widget>
   <widget class="QLabel" name="General4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ramin Leon Neymeyer</string>
    </property>
   </widget>
   <widget class="QLabel" name="General3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Research Group: Hybrid Integration and Sensing</string>
    </property>
   </widget>
   <widget class="QLabel" name="General2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Frutiger LT Com</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deparment: Photonic Components</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="MCCLabel_2">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>500</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Frutiger LT Com</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Manual control</string>
   </property>
  </widget>
  <zorder>MCCLabel</zorder>
  <zorder>MCLabel</zorder>
  <zorder>FocusNTlabel</zorder>
  <zorder>DeinitializeButton</zorder>
  <zorder>MotorControl</zorder>
  <zorder>OutputNTLabel</zorder>
  <zorder>FocusNanotrak</zorder>
  <zorder>OutputNanotrak</zorder>
  <zorder>InputNTLabel</zorder>
  <zorder>InitializeButton</zorder>
  <zorder>TrackAllButton</zorder>
  <zorder>LatchAllButton</zorder>
  <zorder>InputNanotrak</zorder>
  <zorder>OutputNTSerial</zorder>
  <zorder>InputNTSerial</zorder>
  <zorder>MCSerial</zorder>
  <zorder>FocusNTSerial</zorder>
  <zorder>MotorControlChip</zorder>
  <zorder>MCChipSerial</zorder>
  <zorder>frame</zorder>
  <zorder>MoveMotorButton</zorder>
  <zorder>Logo</zorder>
  <zorder>General</zorder>
  <zorder>MCCLabel_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>OutputNTSerial</tabstop>
  <tabstop>InputNTSerial</tabstop>
  <tabstop>FocusNTSerial</tabstop>
  <tabstop>MCSerial</tabstop>
  <tabstop>MCChipSerial</tabstop>
  <tabstop>InputMotor</tabstop>
  <tabstop>OutputMotor</tabstop>
  <tabstop>MiddleStage</tabstop>
  <tabstop>FocusMotor</tabstop>
  <tabstop>InitializeButton</tabstop>
  <tabstop>TrackAllButton</tabstop>
  <tabstop>LatchAllButton</tabstop>
  <tabstop>MoveMotorButton</tabstop>
  <tabstop>DeinitializeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
